README.md

```diff
@@ -36,7 +36,7 @@ The configuration `config.json` contains following properties in `parameters` ke
     - `schema` - string (required)
 - `columns` - array (optional): List of columns to export (default all columns)
 - `outputTable` - string (required): Name of the output table 
-- `incremental` - bool (optional):  Enables [Incremental Fetching](https://help.keboola.com/components/extractors/database/#incremental-fetching)
+- `incremental` - bool (optional):  Enables [Incremental Loading](https://developers.keboola.com/extend/generic-extractor/incremental)
```

Comment:
tady by bylo lepší odkazovat na:
https://help.keboola.com/storage/tables/#incremental-loading

Reply:
diki,upravim to